Practical9

jainamb0907 opened the pull request:
Create a Web page in MVC Project and implement it with the help of JavaScript, and performed Unit Testing.

jainamb0907 committed:
Practical9

create a Web page in MVC Project and implement it with the help of JavaScript, and performed Unit Testing

dilaara-simform commented on Practical9/Practical9/Views/Test2/Index.cshtml:
```diff
@@ -0,0 +1,47 @@
+
+
+
+<script>
```
make use of external JS

dilaara-simform commented on Practical9/Practical9/Views/Shared/_Layout.cshtml:
```diff
@@ -0,0 +1,40 @@
+﻿<!DOCTYPE html>
```
folder structure shall be improved while creating project it should not be complex nesting

dilaara-simform commented on Practical9/Practical9/Controllers/Test1Controller.cs:
```diff
@@ -0,0 +1,18 @@
+
+namespace Practical9.Controllers
+{
+    public class Test1Controller : Controller
```
Controller name should be meaningful

jainamb0907 committed:
Modified changes

Updated Controller Name
Test1Controller- DisplayStringController
Test2Controller- StyleStringController

Added external JavaScript file as pr9.js
